src/common/reducers/game.js

```diff
@@ -77,6 +80,11 @@ export function handleAction(oldState, action) {
     case actions.DESELECT:
       return g.deselect(state, action.payload.player);
 
+    case actions.ADD_CARD_TO_TOP_OF_DECK:
+      // Only to be used in sandbox mode.
+      state.players[action.payload.player].deck.unshift({...action.payload.card, id: id()});
```

Comment:
This is a pretty long line. It could be broken down with two changes.
Deck could get a new method on it `addToTop` and the card definition could be pulled into its own line:

```javascript
const card = {...action.payload.card, id: id()};
const player = action.payload.player;
state.players[player].deck.addToTop(card);
```

Reply:
I'll clean it up a bit.